<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
            background-color:#e2ebf0;          
			border-style: solid;
        }</string>
  </property>
  <widget class="QLineEdit" name="aEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
            background-color:#FFFFFF;          
			border-style: solid;
        }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>元模型名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ackButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.System San Francisco Display</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
            background-color:#fddb92;          
			border-style: solid;
        }</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
